client/my-sites/checkout/checkout-thank-you/jetpack-standalone-activation-instructions.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import ExternalLink from 'calypso/components/external-link';
+import { SelectorProduct } from 'calypso/my-sites/plans/jetpack-plans/types';
+import { getWPORGPluginLink } from './utils';
+
+interface Props {
+	product: SelectorProduct;
+}
+
+const JetpackStandaloneActivationInstructions: React.FC< Props > = ( { product } ) => {
+	const translate = useTranslate();
+
+	const wporgPluginLink = getWPORGPluginLink( product.productSlug );
+
+	return (
+		<>
+			<ol className="licensing-thank-you-manual-activation-instructions__list">
+				<li className="licensing-thank-you-manual-activation-instructions__list-item">
+					<span className="licensing-thank-you-manual-activation-instructions__step-number">1</span>
+					<span>{ translate( 'Login to an existing Wordpress site as an administrator.' ) }</span>
+				</li>
+
+				<li className="licensing-thank-you-manual-activation-instructions__list-item">
+					<span className="licensing-thank-you-manual-activation-instructions__step-number">2</span>
+					<span>
+						{ translate( 'Go to {{strong}}Plugins > Add New{{/strong}} in the admin menu.', {
+							components: { strong: <strong /> },
+						} ) }
+					</span>
+				</li>
+
+				<li className="licensing-thank-you-manual-activation-instructions__list-item">
+					<span className="licensing-thank-you-manual-activation-instructions__step-number">3</span>
+					<span>
+						{ translate(
+							'Search for {{link}}{{strong}}Jetpack %(pluginName)s{{/strong}}{{/link}}, install and activate.',
+							{
+								components: {
+									strong: <strong />,
+									link: (
+										<Button
+											plain
+											href={ wporgPluginLink }
+											target="_blank"
+											rel="noreferrer noopener"
+										/>
+									),
+								},
+								args: { pluginName: product.shortName },
+							}
+						) }
+					</span>
+				</li>
+			</ol>
+
+			<p>
+				<ExternalLink
+					href="https://jetpack.com/support/install-jetpack-and-connect-your-new-plan/"
```

Comment:
Do we need to update this link to plugin-specific installation guide?

For example
https://jetpack.com/support/performance/jetpack-boost/
https://jetpack.com/support/backup/the-jetpack-backup-plugin/getting-started-with-the-jetpack-backup-plugin/

Or we should wait for more detailed support documents?

Reply:
I will be handling it in a separate PR @manzoorwanijk. Need to confirm first with JPOP